lib/Convos/Core/Backend/File.pm

```diff
@@ -80,13 +80,13 @@ sub messages {
   $re = qr{\Q$re\E}i unless ref $re;
   $re = qr/^(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}) (.*$re.*)$/;
 
-  $args{after}  = Time::Piece->strptime($query->{after},  '%Y-%m-%dT%H:%M:%S')if $query->{after};
-  $args{before} = Time::Piece->strptime($query->{before}, '%Y-%m-%dT%H:%M:%S') if $query->{before};
-  # If neither before nor after are defined, search everything before now
-  $args{before} = gmtime if !$args{before} && !$args{after};
   $args{limit}    = $query->{limit} || 60;
   $args{messages} = [];
   $args{re}       = $re;
+  # TODO: What to do when after is after before?
```

Comment:
Yes, of course. You're right. In that case this method should return an empty array of messages. I don't you need to implement any additional check for that.